docs/microservices/logging-monitoring.md

```diff
@@ -96,28 +102,49 @@ Azure Monitor is a managed service, and configuring an AKS cluster to use Azure
 
 Azure Monitor is billed per gigabyte (GB) of data ingested into the service (see [Azure Monitor pricing](https://azure.microsoft.com/pricing/details/monitor/)). At very high volumes, cost may become a consideration. There are many open-source alternatives available for the Kubernetes ecosystem. For example, many organizations use **Fluentd** with **Elasticsearch**. Fluentd is an open-source data collector, and Elasticsearch is a document database that is for search. A challenge with these options is that they require additional configuration and management of the cluster. For a production workload, you may need to experiment with configuration settings. You'll also need to monitor the performance of the logging infrastructure.
 
+### OpenTelemetry
+
+OpenTelemetry is a cross-industry effort to improve tracing by standardizing the interface between applications, libraries, telemetry and data collectors. When using a libray and framework instrumented with OpenTelemetry, most of the work of tracing traditionally system operations is handled by the underlying libraries, including common scenarios like:
+
+- Logging basic request operations: start/exit time, duration
+- Exceptions thrown
+- Context propagation (e.g. sending a correlation ID across HTTP call boundaries)
+
+Instead, the base libraries and frameworks that handle these operations that handele these calls create rich interrlated Span and Trace data structures and propagate these across contexts. Before OpenTelemetry, these were usually just injected as special log messages or as proprietary data structures specific to the vendor building the monitoring tools. OpenTelemetry also encourages a richer instrumention data model than a traditional logging-first approach, and the logs are made more useful since log messages are linked to the Traces and Spans where they were generated. This often makes finding logs associated with a specific operation or request very easy.
```

Comment:
```suggestion
Instead, the base libraries and frameworks that handle these operations create rich interrelated Span and Trace data structures and propagate these across contexts. Before OpenTelemetry, these were usually just injected as special log messages or as proprietary data structures specific to the vendor building the monitoring tools. OpenTelemetry also encourages a richer instrumentation data model than a traditional logging-first approach, and the logs are made more useful since log messages are linked to the Traces and Spans where they were generated. This often makes finding logs associated with a specific operation or request very easy.
```